include/s3select.h

```diff
@@ -505,13 +530,21 @@ struct s3select : public bsc::grammar<s3select>
 
       arithmetic_argument = (float_number)[BOOST_BIND_ACTION(push_float_number)] |  (number)[BOOST_BIND_ACTION(push_number)] | (column_pos)[BOOST_BIND_ACTION(push_column_pos)] |
                             (string)[BOOST_BIND_ACTION(push_string)] |
-                            (cast) |
+                            (cast) | (trim_one) | (trim_two) | (trim_three) |
```

Comment:
use another rule.
trim =  (trim_one) | (trim_two) | (trim_three)

use meaningful name instead of trim_one/two/three